nucliadb/src/nucliadb/search/search/rank_fusion.py

```diff
@@ -62,21 +62,28 @@ def window(self) -> int:
         return self._window
 
     def fuse(
-        self, keyword: Iterable[TextBlockMatch], semantic: Iterable[TextBlockMatch]
+        self,
+        keyword: Iterable[TextBlockMatch],
+        semantic: Iterable[TextBlockMatch],
+        graph: Iterable[TextBlockMatch],
     ) -> list[TextBlockMatch]:
         """Fuse keyword and semantic results and return a list with the merged
         results.
 
         """
-        merged = self._fuse(keyword, semantic)
+        merged = self._fuse(keyword, semantic, graph)
         return merged
 
     @abstractmethod
     def _fuse(
-        self, keyword: Iterable[TextBlockMatch], semantic: Iterable[TextBlockMatch]
+        self,
+        keyword: Iterable[TextBlockMatch],
+        semantic: Iterable[TextBlockMatch],
+        graph: Iterable[TextBlockMatch],
     ) -> list[TextBlockMatch]: ...
 
 
+# DEPRECATED: only used in tests!
```

Comment:
Consider moving it to the test directory then? To some helper in `tests/search/unit` or something?

We could also rewrite the tests to use RRF, but not sure if that makes sense. In any case, moving this out should be easy and it's less code to see here.